CHANGELOG.md

```diff
@@ -18,8 +20,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Changed
 
 - Upgrade `openshift` to `0.8.8`
-- Migrate apps to the new tray package tree
-- Refactor and test the `apps` lookup to support the new tray package tree
+- Migrate apps to the new tray package tree (BC)
```

Comment:
or use the gitmoji for breaking change?

Reply:
Arf, I should have done this in the corresponding commits, my bad. BTW, making a major release implies breaking change(s).